lib/tasks/codereview.rake

```diff
@@ -0,0 +1,11 @@
+require 'pronto'
+
+desc "codereview"
+task :codereview do
+  Pronto.gem_names.each { |gem_name| require "pronto/#{gem_name}" }
+
+  puts "ENV", ENV['PULL_REQUEST_ID']
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.